scripts/validate_docstrings.py

```diff
@@ -471,6 +471,46 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        i = list(self.doc_parameters).index(param)
+        desc_list = [line
+                     # The 'Parameters' value of NumpyDocString._parsed_data is
+                     # a list of tuples. Each tuple represents a parameter
+                     # docstring parameter and has the description of
+                     # the parameter as last element.
+                     for line in self.doc._parsed_data['Parameters'][i][-1]
```

Comment:
Can you use `.desc` here instead of -1? Returns a NamedTuple I think so using attribute is more readable / futureproof

Reply:
Hi @WillAyd thank you for the tip, this looks more clean! I have realised that I was using an older version of numpydoc in my virtualenv. I have updated the code.

Reply:
Hi @WillAyd , did you have a chance to look at this? Please let me know if there is something else I should change.